### ISSUE TITLE
support for additional sourcepath config

### ISSUE THREAD

**Issue by stoicflame** [stoicflame/enunciate#427]
Add support for adding something to the Enunciate sourcepath via Maven/Ant config.

**Comment by nabilzhang** [comment]
I want contribute this, but when I add the sourcepath and test myself, some exception throw,I'm not know well the code.So expecting your success.

**Comment by stoicflame** [comment]
Okay.

So there already _was_ a way to configure the _sourcepath_ by using the Maven dependencies and using the `sourcepath-includes` and `sourcepath-excludes` elements of the plugin configuration. However, this particular issue was opened in relation to #182, which was a case that needed the ability to set the source directories for the Enunciate compiler. 

By default, Enunciate just used the Maven project compiler source directories, but at 52d75a4 I added the ability to use the `sources` plugin configuration to override the compiler source directories with a custom setting.

I found that the Ant task already had the ability to do this.
